misc/services/mobile-webui/mobile-webui-frontend/src/reducers/wfProcesses/index.js

```diff
@@ -91,11 +91,19 @@ export const getStepByIdFromLine = (line, stepId) => {
   return line?.steps?.[stepId];
 };
 
-export const getStepByQRCodeFromActivity = (activity, lineId, qrCode) => {
+export const getNonIssuedStepByQRCodeFromActivity = (activity, lineId, qrCode) => {
   const qrCodeNorm = toQRCodeString(qrCode);
   const line = getLineByIdFromActivity(activity, lineId);
   const steps = getStepsArrayFromLine(line);
-  return steps.find((step) => toQRCodeString(step.huQRCode) === qrCodeNorm);
+  return steps
+    .filter((step) => !(step.qtyIssued > 0 || step.qtyRejected > 0))
```

Comment:
pls extract "(step.qtyIssued > 0 || step.qtyRejected > 0)" to a function, something like isIssued(step)